taxonomy-editor-frontend/src/components/Loader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Box } from "@mui/material";
+import CircularProgress from "@mui/material/CircularProgress";
+
+const Loader = () => {
+  return (
+    <Box
+      sx={{
+        width: "100%",
+        textAlign: "center",
+        py: 10,
```

Comment:
please when replacing the `Loader` component the margins to the places where used, for example, having `py:10` feels like a lot of space in the children or parent list